boot/setup.sh

```diff
@@ -87,6 +89,19 @@ function manta_setup_buckets_api {
         sleep 1
     done
 
+    #
+    # We join the metric ports in a comma-separated list, then add this list as
+    # metricPorts mdata to allow scraping by cmon-agent.
+    #
+    # The metricPorts values are derived from the buckets-api service's "SIZE"
+    # SAPI metadata. We don't need to worry about keeping the metricPorts updated
+    # if this variable changes, because such a change does not affect
+    # already-provisioned zones. This is because electric-moray zones pull the
```

Comment:
Small nit: change "electric-moray" to "buckets-api". This isn't your fault -- I just looked and the muskie code also has this typo . . .

Reply:
ugh I missed that, fixed now.